graphein/ppi/parse_biogrid.py

```diff
@@ -0,0 +1,202 @@
+# %%
+# Graphein
+# Author: Ramon Vinas, Arian Jamasb <arian@jamasb.io>
+# License: MIT
+# Project Website: https://github.com/a-r-j/graphein
+# Code Repository: https://github.com/a-r-j/graphein
+import logging
+from typing import Dict, List, Union
+
+import pandas as pd
+import requests
+
+log = logging.getLogger(__name__)
+
+
+def params_BIOGRID(
+    params: Dict[str, Union[str, int, List[str], List[int]]], **kwargs
```

Comment:
Is `params`' behaviour a bit like the Config objects elsewhere in the library? If so, would it make sense to create a new Config object type?

Reply:
Both options look ok to me. I leave this one to @a-r-j

Reply:
I'm not sure on the right call here. I suspect having it all organised in a config object is cleaner, but I think it might make the lower-level API uglier here - especially if we'd like whether or not a function accepts a config to delineate the two APIs as with the protein graph functions.